src/Transactions/Transfer/TransferAddModalForm.js

```diff
@@ -21,7 +28,43 @@ import {
 
 const TRANSFER_FORM = 'transferForm';
 
-const TransferAddModalForm = ({ handleSubmit, onClose, funds }) => {
+const TransferAddModalForm = ({ budgetFund, handleSubmit, onClose, funds, store, dispatch, change }) => {
+  const formValues = getFormValues(TRANSFER_FORM)(store.getState());
+  const transferFrom = get(formValues, 'fromFundId');
+  const transferTo = get(formValues, 'toFundId');
+  const isTransferFromReqired = has(formValues, 'toFundId') ? transferTo !== budgetFund : false;
+  const validateTransferFrom = isTransferFromReqired ? { validate: validateRequired } : {};
+
+  const getOptionsFrom = () => (
+    (!has(formValues, 'toFundId') ||
+    formValues.toFundId === budgetFund)
+      ? funds
+      : funds.filter(f => f.value === budgetFund)
+  );
+
+  const getOptionsTo = () => (
+    (!has(formValues, 'fromFundId') ||
```

Comment:
do you really need `has` here? IMHO, it's suitable for complex paths, but just for check key in object - it's overkill.
`if ('key' in formValues)`
`formValues.hasOwnProperty('key')`

Reply:
done